style.css

```diff
@@ -102,7 +111,7 @@ h6 {
 }
 .main-container {
     display: grid;
-    grid-template-rows: 100px 170px 20px auto;
+    grid-template-rows: var(--header-height) 170px 20px auto;
```

Comment:
var(--header-height) makes alignment of the header tab easy

Reply:
Also need to start using this myself